e2e-tests/.eslintrc.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
```

Comment:
Why does this not use the main `.eslintrc.js` file? Or does it?

Reply:
Is linting part of CI checks? Does not has to be done in this PR, maybe just add a note to the e2e CI issue.

Reply:
as we want linting for typescript files the rules differ. I think I will look into inheriting `eslintrc` files and check whether they make sense here

Reply:
> Is linting part of CI checks? Does not has to be done in this PR, maybe just add a note to the e2e CI issue.

at least there currently is a command for linting in the ci. But I didn't find where it was actually executed.

Reply:
It seems like it has to be called separately for every workspace. See `check_web` and `check_native`